training/mozilla_voice_stt_training/util/flags.py

```diff
@@ -135,6 +135,7 @@ def str_val_equals_help(name, val_desc):
     # Geometry
 
     f.DEFINE_integer('n_hidden', 2048, 'layer width to use when initialising layers')
+    f.DEFINE_boolean('layer_norm', True, 'wether to use layer-normalization after each fully-connected layer (except the last one)')
```

Comment:
I'm unsure we want to turn that on by default, can we make it `False` here?

Reply:
yes, it was default to true for our application